src/Microsoft.DotNet.Build.Tasks/PackageFiles/versioning.targets

```diff
@@ -115,7 +115,7 @@
       <NativeVersionLines Include="#endif" />
     </ItemGroup>
 
-    <MakeDir Condition="!Exists('$(NativeVersionFileDir)')" Directories="$(NativeVersionFileDir)" />
```

Comment:
Why are you removing this condition?

Reply:
MakeDir doesn't throw if it already exists. The condition isn't necessary.